### PR TITLE
feat(ImportUshahidiV2): add parametrized imports and other fixes

### PR DESCRIPTION
**Pull request by tuxpiper**
This pull request makes the following changes:
- on top of #3466 , add feature so that the ImportUshahidiV2 command can take an extra parameter to a file with operation specific parameters
- the currently available options in the parameter file are to specify mappings for categories, forms and fields

Test checklist:
- [ ] Automated tests still broken. WIP.
- [ ] Successfully run the import for one of our clients, which requires use of pre-established mappings for categories, forms and fields

- [ ] I certify that I ran my checklist

Fixes ushahidi/platform# .

Ping @ushahidi/platform

### CODE REVIEW

**Comment by tuxpiper** [comment]
@rowasc @Angamanga  we'll be reviewing this one over a call, sending invites

**Comment by tuxpiper** [comment]
@rowasc  Since we talked about this on a call and I think the only sticking point was documentation, suggesting to merge this down, not to develop, but to the whole migration-tool PR/branch ( #3466 )  . Thoughts?

In parallel will be opening a draft on gitbook.

I hope we can merge the whole V2-V3 thing in the next release or the other

File: src/Core/Tool/ManifestLoader.php [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+    public function loadManifestFromFile(string $path)
+    {
+        // Check that file exists
+
+        // Check that the size of the file is not crazy
+
+        // Get file contents
+        $contents = file_get_contents($path);
+        if (!$contents) {
+            // raise exception
```
[the remark this answers is not in the document]
Comment (rowasc) [reply src/Core/Tool/ManifestLoader.php]: @tuxpiper the only other points I see are a few of this TODOs scattered around the code?  not sure if you want to handle them pre-merge or after you merge to the migration branch (whatever works for you!)

[the remark this answers is not in the document]
Comment (tuxpiper) [reply src/Core/Tool/ManifestLoader.php]: yep, shall do an additional pass / polish before merging to develop. There are also the tests to fix

**Pull request merged**